.agents/skills/brev-cli/reference/commands.md

```
@@ -466,6 +466,50 @@ brev port-forward my-instance -p 8080:8080
 ```
 
+### brev ports
+List Skybridge-managed HTTP applications and raw network port mappings for a
+managed instance or registered compute node.
+
+```bash
+brev ports <instance-or-node> [flags]
+```
+
+**Flags:**
+| Flag | Description |
```

markdownlint-cli2 (0.23.2)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)